include/drawtypes/label.hpp

```diff
@@ -51,27 +53,28 @@ namespace drawtypes {
     size_t m_maxlen{0_z};
     bool m_ellipsis{true};
 
-    explicit label(string text, int font) : m_font(font), m_text(text), m_tokenized(m_text) {}
+    explicit label(string text, int font) : m_font(font), m_text(std::move(text)), m_tokenized(m_text) {}
     explicit label(string text, string foreground = ""s, string background = ""s, string underline = ""s,
-        string overline = ""s, int font = 0, struct side_values padding = {0U,0U}, struct side_values margin = {0U,0U},
-        size_t maxlen = 0_z, bool ellipsis = true, vector<token>&& tokens = {})
-        : m_foreground(foreground)
-        , m_background(background)
-        , m_underline(underline)
-        , m_overline(overline)
+        string overline = ""s, int font = 0, side_values padding = {0U, 0U}, side_values margin = {0U, 0U},
+        size_t maxlen = 0_z, bool ellipsis = true, vector<token>&& tokens = {}, vector<string>&& token_whitelist = {})
+        : m_foreground(move(foreground))
+        , m_background(move(background))
+        , m_underline(move(underline))
+        , m_overline(move(overline))
         , m_font(font)
         , m_padding(padding)
         , m_margin(margin)
         , m_maxlen(maxlen)
         , m_ellipsis(ellipsis)
-        , m_text(text)
+        , m_text(move(text))
         , m_tokenized(m_text)
-        , m_tokens(forward<vector<token>>(tokens)) {
```

Comment:
Forward should only be used on universal references, here it is a rvalue so move should be used.
(See https://en.cppreference.com/w/cpp/utility/forward or if you want to look deeper into that https://isocpp.org/blog/2012/11/universal-references-in-c11-scott-meyers)